src/JJMasterData.Core/DataManager/Exports/DataExpReporter.cs

```diff
@@ -31,7 +32,6 @@ public int TotalRecords
     private void UpdatePercentage()
     {
         if (TotalRecords > 0 && TotalProcessed > 0)
-            Percentage = (int)((TotalProcessed / (double)TotalRecords) * 100);
-
+            Percentage = (int)Math.Round((TotalProcessed / (double)TotalRecords) * 100);
```

Comment:
The error was here, can't repro the error in Example environment, I will try at the real Development website now. Unit tests catched the error.